portfolio/src/main/webapp/scripts/templatemo-script.js

```diff
@@ -46,37 +46,39 @@ $(document).ready(function () {
     document.querySelector('.tm-current-year').textContent = new Date().getFullYear();
 });
 
-var pictures = new Array("images/apollo.jpg", "images/bob.jpeg", "images/boomer.jpg", "images/drift.jpg",
-                        "images/filbert.jpg", "images/kiki.jpg", "images/lolly.jpg", "images/punchy.jpg",
-                        "images/rosie.jpg");
+var pictures = new Array(new Array("images/blathers-negative.png", "images/isabelle-negative.png"), 
+                        new Array("images/blathers-neutral.png", "images/celeste-neutral.png", "images/isabelle-neutral.png"), 
+                        new Array("images/blathers-positive.png", "images/celeste-positive.png", "images/isabelle-positive.png"));
 
 /**
  * Fetch the comment data in Json format and create a comment box in the comments section for each comment.
  */
 async function fetchJson() {
     const response = await fetch('/data');
-    const json = await response.json();
+    const jsonData = await response.json();
 
     var template = document.querySelector('#comment-template');
     var commentBox = template.content.querySelector('div');
     var temp;
 
-    json.forEach(comment => {
-        temp = modifyTemplate(commentBox, comment);
-        document.querySelector('#comments').appendChild(temp);
+    Object.keys(jsonData).forEach(function(key) {
+        var text = jsonData[key]['text'];
+        var score = jsonData[key]['score'];
+        temp = modifyTemplate(commentBox, text, score);
+        document.querySelector('#comment-list').appendChild(temp);
     });
 }
 
 /**
  * Replace the icon and text of the template comment box with a random icon and the comment.
  */
-function modifyTemplate(commentBox, comment) {
+function modifyTemplate(commentBox, text, score) {
     const tempBox = document.importNode(commentBox, true);
 
-    var randomNum = Math.floor(Math.random() * pictures.length);
-    tempBox.querySelector('#comment-icon').src = pictures[randomNum];
+    var randomNum = Math.floor(Math.random() * pictures[score].length);
```

Comment:
this line of code is a bit smelly.
lets create an enum of sentiment (positive & negative) and a function to convert the score to those enum.
next, we can either:
- use switch case for all the enum in the sentiment
- change pictures constant to a dictionary with enum as the key and image as the value

Reply:
I've made the change